### DISCUSSION TITLE
[Feature] Mobile: Filter by sync state

### DISCUSSION THREAD

**Discussion by jordanlambrecht** [immich-app/immich#8777] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I really want to only see images that are synced rather than see all images on my phone regardless of sync state. Would it be difficult to implement a toggle for this?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2155
